<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StemDetectorFrame</class>
 <widget class="QWidget" name="StemDetectorFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTableWidget" name="tblDetectors">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Inner</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Outer</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>x centre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>y centre</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="tabPanel" name="tPanel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="vLayout">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtName">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Inner radii</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtInner">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Outer radii</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtOuter">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>x centre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtCentreX">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>y centre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtCentreY">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="vSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDelete">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tabPanel</class>
   <extends>QTabWidget</extends>
   <header>controls/tabpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
